document/core/syntax/instructions.rst

```diff
@@ -47,15 +57,27 @@ These operations closely match respective operations available in hardware.
    \begin{array}{llcl}
    \production{width} & \X{nn}, \X{mm} &::=&
      \K{32} ~|~ \K{64} \\
+   \production{simdwidth} & \X{sss} &::=&
```

Comment:
This should be `nnn`. But is it even worth introducing it as long as we only have one SIMD type?

Reply:
Changed to nnn. Probably not worth it, inlined it to be v128 in all the necessary instructions.